Source/BSN.Commons.S3.Net/BSN.Commons.S3.Net.csproj

```diff
@@ -0,0 +1,102 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <ProjectGuid>{5F798AA1-1A65-4C9F-8415-3100C141DD87}</ProjectGuid>
+    <OutputType>Library</OutputType>
+    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <RootNamespace>BSN.Commons.S3.Net</RootNamespace>
+    <AssemblyName>BSN.Commons.S3.Net</AssemblyName>
+    <TargetFrameworkVersion>v4.7.2</TargetFrameworkVersion>

```

Comment:
Why we need two project for .Netframework and Core?

As I know, S3 support both of them, so why do not create one project with .NET Standard 2.0 for S3 implementation?

Reply:
https://www.nuget.org/packages/AWSSDK.S3

Reply:
I have argued the issues with you and we decided to accept the issue with .netframework.

Reply:
Now it's a part of the BSN.Commons.